### ISSUE TITLE
Learning from synthetic data generated with GRADE. (arXiv:2305.04282v2 [cs.CV] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#26216]
Learning from synthetic data generated with GRADE. (arXiv:2305.04282v2 [cs.CV] UPDATED)<br>
https://ift.tt/KX5xl4j<br>
<p>Recently, synthetic data generation and realistic rendering has advanced
tasks like target tracking and human pose estimation. Simulations for most
robotics applications are obtained in (semi)static environments, with specific
sensors and low visual fidelity. To solve this, we present a fully customizable
framework for generating realistic animated dynamic environments (GRADE) for
robotics research, first introduced in [1]. GRADE supports full simulation
control, ROS integration, realistic physics, while being in an engine that
produces high visual fidelity images and ground truth data. We use GRADE to
generate a dataset focused on indoor dynamic scenes with people and flying
objects. Using this, we evaluate the performance of YOLO and Mask R-CNN on the
tasks of segmenting and detecting people. Our results provide evidence that
using data generated with GRADE can improve the model performance when used for
a pre-training step. We also show that, even training using only synthetic
data, can generalize well to real-world images in the same application domain
such as the ones from the TUM-RGBD dataset. The code, results, trained models,
and the generated data are provided as open-source at
https://ift.tt/Hrzp2bo.
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
